cnv/cnv_users_guide/cnv-fencing-unhealthy-nodes.adoc

```diff
@@ -0,0 +1,27 @@
+[id="cnv-fencing-unhealthy-nodes"]
+= Fencing unhealthy nodes
+include::modules/cnv-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: cnv-fencing-unhealthy-nodes
+toc::[]
+
+Fencing is the process of isolating a failed node to protect a cluster and its resources.
+A node is considered 'fenced' when it can no longer access shared resources in order
```

Comment:
Nit: I'm not sure if "in order for those resources to be used elsewhere" is necessary.